workflow/lib/idds/iworkflow/work.py

```diff
@@ -779,10 +779,9 @@ def load(self, func_name):
 
         :raise Exception
         """
-        os.environ['IDDS_IWORKFLOW_LOAD_WORK'] = 'true'
-        func = super(Work, self).load(func_name)
-        del os.environ['IDDS_IWORKFLOW_LOAD_WORK']
-
+        import idds.common.utils as idds_utils
```

Comment:
move the import to the top. Normally the import should be on the top, except importing some clients.